lighthouse-core/gather/gather-runner.js

```diff
@@ -165,6 +166,20 @@ class GatherRunner {
     } else if (mainRecord.hasErrorStatusCode()) {
       errorDef = {...LHError.errors.ERRORED_DOCUMENT_REQUEST};
       errorDef.message += ` Status code: ${mainRecord.statusCode}.`;
+    } else if (!mainRecord.finished) {
+      // could be security error
```

Comment:
I think this may be just certificate errors. Seems like most of the connection errors are `mainRecord.failed === true`.